### DISCUSSION TITLE
Time periods instead of times

### DISCUSSION THREAD

**Discussion by InnovateMobility** [FlowmapBlue/FlowmapBlue#65] [Q&A]
Hello, is it possible to input a time period instead of a specific time?  For example, I have some data that has geo points between midnight and 6am, 601am to 10am, etc etc.  I don't have specific hours for each point.

Thanks much!

**Comment by ilyabo** [comment]
Hello. You should be able to just use the start time of each time period. Alternatively, you can partition you data into multiple sheets like is done [in this map](https://flowmap.blue/1mK1ZMxNmGtSSxMhtoKO5h7nxyDMXFC_3_u4eo4rtucg/e3131e8?v=45.513266%2C-73.573425%2C12.18%2C0%2C0&a=0&as=1&b=1&bo=75&c=1&ca=1&d=1&fe=1&lt=1&lfm=ALL&col=Teal&f=45). Check the `flows.sheets` property description in the [template spreadsheet](https://docs.google.com/spreadsheets/d/1aEgwtGUGc0TdnsO0jIm50hshCZ-m4DHms3P0Qq9IYdA/edit#gid=1438429083).
